### ISSUE TITLE
LR Scheduler cycles and power interface suggestion

### ISSUE THREAD

**Issue by abetteridea** [bmaltais/kohya_ss#1217]
I just discovered GUI textboxes for 2 parameters that I have been setting manually with _lr_scheduler_num_cycles_ and _lr_scheduler_power_ in **Optimizer Extra arguments** textbox.

![image](https://github.com/bmaltais/kohya_ss/assets/67851516/b019af32-5147-4a3e-a445-0a5f1a5cf977)

Turns out they are buried in **Advanced Configuration**. It makes sense to an extent, but wouldn't it be more logical to have them next to LR settings here?

![image](https://github.com/bmaltais/kohya_ss/assets/67851516/293e9986-2bdf-4e53-8d6b-03d52bbbbca7)

An even more logical implementation would be to only show these settings when _Cosine with restarts_ or _Polynomial_ are selected in LR Scheduler dropdown.

Actually, as far as I understand, _lr_scheduler_num_cycles_ only applies to _**Cosine with restarts**_ while _lr_scheduler_power_ only applies to _**Polynomial**_. So I guess at least placeholder texts need to be changed.

**Comment by bmaltais** [comment]
Interesting suggestion. The GUI parameters is a bit of a hot mess as I keep adding options as kohyaA_ss add them to his code... And I don't think the logical layout is great. The GUI would benefit from a full redo... I have not found the strength to start work on this yet :-)

I will what I can do about those 2 parameters as I think they would be better grouped the way you describe them... But the more I start to move parameters outside Advanced the more cluttered the main parameter section becomes... Not sure how I can do both while maintaining something that is not too overwhelming for users.

**Comment by abetteridea** [comment]
I totally understand, of course! That's why I suggest to only show these parameters when _Cosine with restarts_ or _Polynomial_ are picked from the dropdown menu. This way we don't clutter the default main section view.
